### DISCUSSION TITLE
With Docker installation, how to change the reset_url

### DISCUSSION THREAD

**Discussion by cvvergara** [WeblateOrg/weblate#10042] [Q&A]
Hello,

We have a docker installation, and the login is with LDAP.

The "Forgot your password?" link goes to: `https://https://weblate.my.org/accounts/reset/`
But the location of the of the LDAP registration and edition is located elsewhere like `https:/ldap.my.org/`

Looking at this [line](https://github.com/WeblateOrg/weblate/blob/ac50f44f72e75e37ec51f27ea8a0667808b32238/weblate/templates/accounts/login.html#L58) **reset_url** is the one I need to change to go to the real LDAP registration/reset page.

This documentation does not show how to go to the LDAP registration/reset page
https://docs.weblate.org/en/latest/admin/auth.html#ldap-authentication

How can I solve the problem?
Regards

**Comment by nijel** [comment]
There is currently no way to customize reset url, I've created https://github.com/WeblateOrg/weblate/issues/10101 for you to track this feature request.

**Reply by strk** [reply]
It would make sense not only to customize the reset url but also the text itself, as if you dont have an account  yet it's weird that you have to click "forgot password?"... a "sign up" would be more appropriate

**Comment by cvvergara** [comment]
Workaround as follows:
On the nginx configuration file for weblate, add a redirection
```
server {
   ....
   rewrite ^/accounts/reset$ https://id.my.org redirect;
   ....
}
```
